how_to/contribute_to_the_bpd_community.md

```diff
@@ -1 +1,87 @@
 # How to Contribute to the BPD Community
+Contributing to open source can be a rewarding experience, and getting started is easier than you might think. Here’s a beginner-friendly guide to help you navigate your way into the open source world for Black Python Dev repo:
+The repo link: https://github.com/BlackPythonDevs/blackpythondevs
+
+### 1. **Understand What Open Source Is**
+
+Open source software is developed collaboratively and made available to anyone to use, modify, and distribute. Contributions can range from fixing bugs and adding features to improving documentation.
+
+### 2. **Find a Project**
+
+
+
+#### **Look for Good First Issues**
+
+Many projects label issues that are suitable for beginners. Search for labels like “good first issue,” “beginner,” or “help wanted.”
+You can use this link to see some of the available issues: https://github.com/BlackPythonDevs/blackpythondevs/issues
+
+
+#### **Read the Documentation**
+
+- **README.md**: The primary document that usually provides an overview, installation instructions, and usage guidelines.  Read the read me first:
```

Comment:
```suggestion
- **README.md**: The primary document that usually provides an overview, installation instructions, and usage guidelines.  Read the README first:
```

Reply:
Does it mean that it will be merged now? since you have fixed some of the typos

Reply:
@CoderBlee if you are happy with the suggestions you can commit them and let us know.

Reply:
@ronnzw  the one with conflicts, I do not know how to solve it anymore, also how do I save the new changes without is being flagged